### ISSUE TITLE
web: not able to run on web 

### ISSUE THREAD

**Issue by venkata-reddy-dev** [rydmike/flex_scaffold_exp#4]
Issue :

 ../../../../.pub-cache/hosted/pub.dev/web-0.3.0/lib/src/dom/webrtc.dart:414:33: Error: The type 'JSObject' can't be used as supertype.
class RTCCertificate implements JSObject {}

![Screenshot 2024-02-29 at 8 24 58 AM](https://github.com/rydmike/flex_scaffold_exp/assets/32235964/4af08219-7c60-423a-ab6e-49470eaed979)

**Comment by rydmike** [comment]
Yes it is just a version bump on some old cached deps. It will be fixed when I work on the new Web demo app. The current version will not be used at all when this released.
